crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -462,9 +465,17 @@ where
         bin_key: BPI::BinKey,
         (entity, main_entity): (Entity, MainEntity),
         input_uniform_index: InputUniformIndex,
-        phase_type: BinnedRenderPhaseType,
+        mut phase_type: BinnedRenderPhaseType,
         change_tick: Tick,
     ) {
+        // If the user has overridden indirect drawing for this view, we need to
+        // force the phase type to be batchable instead.
+        if let GpuPreprocessingMode::PreprocessingOnly = self.gpu_preprocessing_mode {
```

Comment:
Why are these if let? Why not just compare the variant with == or use matches!()?

Reply:
Mild preference to just use == but I don't particularly mind this style.